tests/test_cli_loads.py

```diff
@@ -43,6 +43,62 @@ def test_init_command(self):
 
         # Clean up
         shutil.rmtree(test_dir)
+    
```

Comment:
please add/merge these into the file too

```python
    def setUp(self):
        self.TEST_DIR = Path("test_project")

    def run_cli(self, *args, cwd=None):
        """Helper method to run the CLI with arguments."""
        result = subprocess.run(
            [*self.CLI_ENTRY, *args],
            capture_output=True,
            text=True,
            cwd=cwd
        )
        return result
```

Reply:
Done.